AWS-CodePipeline/sam-monorepo-pipeline-template/questions.json

```diff
@@ -0,0 +1,214 @@
+            "isRequired": true,
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "test_stage_name"
+                    },
+                    "region"
+                ]
+            }
+        },
+        {
+            "key": "message_test_stage_configured",
+            "question": "Stage 1 configured successfully, configuring stage 2.\n",
+            "kind": "info"
+        },
+        {
+            "key": "message_list_stage_names_prod",
+            "question": {
+                "keyPath": [
+                    "stage_names_message"
+                ]
+            },
+            "kind": "info"
+        },
+        {
+            "key": "prod_stage_name",
+            "question": "Select an index or enter the stage 2's configuration name (as provided during the bootstrapping)",
+            "allowAutofill": true,
+            "isRequired": true
+        },
+        {
+            "key": "prod_stack_name",
+            "question": "What is the sam application stack name for stage 2?",
+            "isRequired": true,
+            "default": "sam-app-prod"
+        },
+        {
+            "key": "prod_pipeline_execution_role",
+            "question": "What is the pipeline execution role ARN for stage 2?",
+            "isRequired": true,
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "prod_stage_name"
+                    },
+                    "pipeline_execution_role"
+                ]
+            }
+        },
+        {
+            "key": "prod_cloudformation_execution_role",
+            "question": "What is the CloudFormation execution role ARN for stage 2?",
+            "isRequired": true,
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "prod_stage_name"
+                    },
+                    "cloudformation_execution_role"
+                ]
+            }
+        },
+        {
+            "key": "prod_artifacts_bucket",
+            "question": "What is the S3 bucket name for artifacts for stage 2?",
+            "isRequired": true,
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "prod_stage_name"
+                    },
+                    "artifacts_bucket"
+                ]
+            }
+        },
+        {
+            "key": "prod_image_repository",
+            "question": "What is the ECR repository URI for stage 2?",
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "prod_stage_name"
+                    },
+                    "image_repository"
+                ]
+            }
+        },
```

Comment:
imagine repository questions were asked but I don't see them being used in the pipeline.yaml file

Reply:
I removed them because in our example use case they weren't being used. I'll add it back shortly. Thanks for pointing it out!

Reply:
I've added it back, but to simplify the CF stack, I've added it only to `samconfig.toml`. It should have the exact same result. See `pipeline/samconfig.toml` and `pipeline/buildspec_build_package.yml`